### ISSUE TITLE
Preview from local machine should not appear in Activity tab

### ISSUE THREAD

**Issue by scav** [pulumi/service-requests#189]
When a developer wants to preview their code change before submitting a PR, they can do `pulumi preview` on the stack in question. However, this ends up under the stack in the Pulumi Service as a logged preview. 

Ideally simply running preview outside of a pipeline should be possible to without cluttering the Activity tab of the stack. To maintain insight into who did what, previews that potentially share information could be audit logged, but not added to the activity tab. E.g. a `pulumi preview --local` or some other mechanic to signal that we are simply checking if everything works.

## Affected feature
Pulumi CLI and Pulumi Service

**Comment by EvanBoyle** [comment]
@scav thanks for the suggestion. the current design is that all previews are shown until an update is run, then we collapse down those previews only showing the most recent before the update (the assumption being that the last preview matches the update). 

We do want every operation to be visible and sharable within the console. We don't really distinguish between operations run within CI and locally (except for some iconography). If a local preview fails it should be easy to share a link to the output with your colleagues. 

Are you running so many previews that this becomes a major problem?

**Comment by scav** [comment]
I agree with the sentiment that every operation should be visible, I don't belive a preview necessarily qualifies as an operation (on state) in itself. Is there a limit to how many previews are shown? If someone were to run ten of them, would we eventually end up with a lower number of previews visible in the console than what actually ran?

We are not running so many previews that it has become a major problem, I just would prefer it be possible to opt out of this as it potentially adds a lot of noise to something that has no real impact on state. That is why I suggested a flag (which could probably be given a much better name) to optionally opt out of this feature if you wanted to preview a lot.

I guess since it is by design, it can be closed, and we just adapt our workflow to it.
